api-test/UnitTest/Dto/BrandDTO.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using api.DTOs;
+using Xunit;
+
+namespace api_test.UnitTest.DTOs
+{
+    public class BrandDTOTest
+    {
+        [Fact]
+        public void BrandDTOProperties_ShouldWorkCorrectly()
```

Comment:
hmm you are not calling any functionality here, you are only putting the expected result and comparing, why?